build.sh

```diff
@@ -4,12 +4,9 @@
 # and builds AppImage
```

Comment:
Can we leave `build.sh` as is for a while, and have `build-with-cmake.sh`in parallel for some time? I'd like to have something to compare against, at least temporarily. I'd like to keep `build.sh` in a working state at least for some weeks

Reply:
In fact, can't we make a shell script like this entirely redundant when using CMake? Can't CMake do all of this on its own when invoked with no additional parameters?

Reply:
This shell script is redundant, yes. But it was you who asked for it to stay in place so that there's a single build script you can use to build this repository, without having to use CMake directly.

Reply:
With "stay" I meant "stay without being converted to CMake stuff", of course. So that I still have a version that I understand and can compare the new CMake based version with.

As for the CMake based version, I don't think it should need this script anymore.

Reply:
We had it "stay" for months now, after merging the first bits of CMake code. You can always find it in the Git history, but maintaining it any further makes absolutely no sense to me.